gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -107,6 +112,26 @@ export default function NxFormLayoutExample() {
           </select>
         </label>
       </div>
+      <fieldset className="nx-fieldset">
+        <legend className="nx-legend">
+          <span className="nx-legend__text">
+            Here's what you can do: {isWarpOn && 'Warp drive started'}
+            {isKrakenOut && ' The Kraken is out!'} {isShapes && ' Shapes are allowed'}
```

Comment:
A label/legend that changes based on on the form fields isn't really a pattern that we have or, I would think, want to encourage.

Reply:
> A label/legend that changes based on on the form fields isn't really a pattern that we have or, I would think, want to encourage.

Hah, no: https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/ab78e6c29549ead547fdc4d8df713039c841aad8